Documentation/report.md

```diff
@@ -292,3 +292,140 @@ A report can have the following states:
 
 
 [rfc3339]: https://tools.ietf.org/html/rfc3339#section-5.8
+
+## Roll-Up Reports
```

Comment:
I like this. I think we'll want more details on the purpose and reason for why you want this and potentially a few more examples. I'm thinking perhaps we do a few things to make that easier to manage:

- We should probably add some built-in aggregated report queries. We can start with namespace for memory & cpu. 
- We can include one inline-version like you have here in the documentation to make it very clear what's happening.
- Perhaps this section could just link to a separate "guide" on roll-up reports. It's kind of a big concept and may deserve it's own document. What do you think?
- We should fix up the underlying issues you've discovered and then remove documentation about these work-arounds for the double-counting, and fix up these queries names.